services/web/server/tests/unit/with_dbs/02/test_projects_crud_handlers__clone_in_workspace_and_folder.py

```diff
@@ -0,0 +1,141 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=too-many-arguments
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+from copy import deepcopy
+from typing import Any, Iterator
+
+import pytest
+import sqlalchemy as sa
+from aiohttp.test_utils import TestClient
+from models_library.api_schemas_webserver.projects import ProjectGet
+from models_library.folders import FolderID
+from models_library.projects import ProjectID
+from models_library.workspaces import WorkspaceID
+from pytest_simcore.helpers.webserver_login import UserInfoDict
+from pytest_simcore.helpers.webserver_parametrizations import MockedStorageSubsystem
+from servicelib.aiohttp.long_running_tasks.client import long_running_task_request
+from simcore_postgres_database.models.folders import folders
+from simcore_postgres_database.models.workspaces import workspaces
+from simcore_service_webserver.db.models import UserRole
+from simcore_service_webserver.folders._folders_api import create_folder
+from simcore_service_webserver.projects._folders_api import move_project_into_folder
+from simcore_service_webserver.projects.models import ProjectDict
+from simcore_service_webserver.workspaces._workspaces_api import create_workspace
+from yarl import URL
+
+
+@pytest.fixture
+async def create_workspace_and_folder(
+    client: TestClient, logged_user: UserInfoDict, postgres_db: sa.engine.Engine
+) -> Iterator[tuple[WorkspaceID, FolderID]]:
+    workspace = await create_workspace(
+        client.app,
+        user_id=logged_user["id"],
+        name="a",
+        description=None,
+        thumbnail=None,
+        product_name="osparc",
+    )
+
+    folder = await create_folder(
+        client.app,
+        user_id=logged_user["id"],
+        name="a",
+        parent_folder_id=None,
+        product_name="osparc",
+        workspace_id=workspace.workspace_id,
+    )
+
+    yield (workspace.workspace_id, folder.folder_id)
+
+    with postgres_db.connect() as con:
+        con.execute(folders.delete())
+        con.execute(workspaces.delete())
+
+
+@pytest.fixture
+def fake_project(
+    fake_project: ProjectDict,
+    workbench_db_column: dict[str, Any],
+    create_workspace_and_folder: tuple[WorkspaceID, FolderID],
+) -> ProjectDict:
+    # OVERRIDES user_project
```

Comment:
I am not sure that comment actually fits what you are doing here.
you override fake_project not user_project.